config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```diff
@@ -61,6 +61,8 @@ coredns_additional_error_config: |
 
 kube_profiling: false
 
+csi_snapshot_controller_enabled: true
```

Comment:
Will this also create the VolumeSnapshotClass? Or only the controller? Same with upcloud

Reply:
This will only create the controller. The volumesnapshotclass is created when cinder_csi is enabled, which seems a little weird. I would want volumesnapshotclass created when its controller is enabled. See [here](https://github.com/elastisys/kubespray/blob/81446462cf0bb04411ee7de1c734c8b7559336e0/roles/kubernetes-apps/snapshots/meta/main.yml) and [here](https://github.com/elastisys/kubespray/blob/81446462cf0bb04411ee7de1c734c8b7559336e0/roles/kubernetes-apps/snapshots/cinder-csi/templates/cinder-csi-snapshot-class.yml.j2)

Do we want to fix that in kubespray ? 

In UpCloud's case, a volumesnapshotclass is created when snapshotcontroller is enabled.

Reply:
Alright, so the volumesnapshotclass should already be installed in all our cluster already then? I'm fine with leaving it tbh.

Reply:
Yes, they're already installed in our clusters. I'll resolve this then.